<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkupWindow</class>
 <widget class="QWidget" name="MarkupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>724</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <item row="1" column="0">
      <widget class="QCustomPlot" name="plot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>8</horstretch>
         <verstretch>8</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>8</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="spacing">
           <number>10</number>
          </property>
          <item row="3" column="1">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Marks:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QListView" name="listView_markups">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>10</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="AudioFileSelector" name="audio_file_selector" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>10</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="pushButton_save">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="AudioViewModeSelector" name="audio_view_mode_selector" native="true">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="GraphControls" name="controls" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">ui/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphControls</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/graph_controls.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AudioFileSelector</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/audio_file_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AudioViewModeSelector</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/audio_view_mode_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
